tests/test_orientation.py

```diff
@@ -72,8 +82,13 @@ def approx_ang(expected_degrees):
     def assert_angle(angle_radians, expected_degrees, message):
         expected_degrees = approx_ang(math.degrees(expected_degrees))
         # Check both +0 and +360 so approx can cover the jump between -180 and 180
-        assert math.degrees(angle_radians) == expected_degrees or math.degrees(angle_radians) + 360 == expected_degrees, \
-            message
+        if allow_wrapping:
```

Comment:
Conditionals in tests are a bad idea; they usually indicate that the test is trying to test too many things and should be broken apart. Tests should be so simple that they are obviously correct.